packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeModal/StakeModal.tsx

```diff
@@ -23,12 +26,15 @@ export const StakeModal = ({ onCancel }: StakeModalModalProps) => {
     return (
         <StakeEthFormContext.Provider value={stakeEthContextValues}>
             <NewModal
-                size="tiny"
+                size="large"
                 heading={<Translation id="TR_STAKE_ETH" />}
                 onCancel={onCancel}
                 bottomContent={<StakeButton />}
```

Comment:
Please remove the isFullWidth prop from the button.

Reply:
removed from staking and unstaking modals, https://github.com/trezor/trezor-suite/pull/14713/commits/aa21e6e49b2370a3e9bbb5035db474d51a1e8519